modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -616,6 +619,32 @@ Given the size of your ownCloud filesystem, this may take some time to complete.
 However, if your filesystem is quite small, then it will complete quite quickly. 
 The `-y` switch can be supplied to automate acceptance of user input.
 
+==== hsmdaemon
+
+`encryption:hsmdaemon` provides the ability to interact with an {hsm-overview-url}[hsm (Hardware Security Module)] which is integrated with ownCloud.
+
+NOTE: This command requires ownCloud to be integrated with xref:configuration/server/security/hsmdaemon/index.adoc[hsmdaemon], which requires ownCloud Enterprise Edition >= 10.2.
+For more information about hsmdaemon, please contact {owncloud-support-url}[ownCloud support].
+
+===== Usage
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} encryption:hsmdaemon [options]
+----
+
+===== Options
+
+[width="100%",cols="40%,60%",]
+|===
+| `--export-masterkey` 
+| Export the private master key, base64 encoded.
```

Comment:
This is too little information yet. 
- Does it export the private master key from the HSM? Is this even possible?
- Is it another private master key in the instance? 
- Do I have to worry about security?

Reply:
Thanks for the feedback.

Reply:
No it does not export the master key from hsm. Its the master key from ownCloud's instance getting exported. To be precise the file residing in `data/files_encryption/OC_DEFAULT_MODULE/master_*.privateKey`.